docs/ADVIES_LEEFGEBIEDEN.md

```diff
@@ -0,0 +1,48 @@
+
+
+## Bronnen
+We baseren ons voor het onderstaande op:
+- Het *Methodendocument begrenzing en afbakening LG-typen M17, versie 21-09-2016*. Dit document is een conceptversie, en niet landelijk officieel vastgesteld. Het document is daarom niet online beschikbaar.
+- De *Toelichting Leefgebiedenkaart Dwingelderveld T0, versie 07-dec-2023 (N2K_LK_30_Dwingelderveld_T0_231207)*, opgemaakt door Prolander, dat voortborduurt op het Methodendocument. Ook dit document is niet online beschikbaar.
+- [Rapport Sovon Leefgebiedenkaarten N2000 en PAS](https://www.bij12.nl/wp-content/uploads/2023/11/Rapport-Sovon-2016-21-Leefgebiedenkaarten-N2000-PAS.pdf)
+
+
+## Leefgebieden VHR-soorten en LG-typen
+Habitattypen en LG-typen vormen samen de leefgebieden voor VHR-soorten. LG-typen vormen een aanvulling op habitattypen; een karteervlak(deel) heeft nooit tegelijkertijd een habitattype én een LG-type, deze sluiten elkaar uit.
+
+Het methodendocument definieert 14 LG-typen. Voor ieder LG-type beschrijft het document:
+- welke beschermde VHR-soorten er leven;
+- op welk(e) habitattype(n) dit LG-type een aanvulling is;
+- welke plantengemeenschappen (i.e. VvN-typen) er tot het type behoren;
+- in welke FGR gebieden de plantengemeenschap moet liggen om tot het type te behoren. Deze voorwaarde geldt in feite als een beperkend criterium;
+- aanvullende abiotische en ruimtelijke criteria. Deze criteria zijn met name relevant voor aquatische typen. Methodendocument: *"In het geval van terrestrische vegetaties zijn plantengemeenschappen [...] veelal wel als zuivere bron bruikbaar."*
+
+N.B.: De plantengemeenschappen uit het Methodendocument zijn later door Prolander aangepast. Ook zijn er door Prolander voor het maken van leefgebiedenkaarten voor Dwingelderveld extra LG-typen aan toegevoegd (LG4010 en LG4030).
+
+
+## Stappenplan automatiseren LG-typenkaart veg2leef
+Veg2hab kan met kleine aanpassingen ook gebruikt worden voor het maken van LG-typekaarten:
+- Voor het inlezen van vegetatiekarteringen kan `stap 1` uit veg2hab onveranderd gebruikt worden. Het methodendocument definieert LG-typen op basis van VvN-codes, stap 1 van veg2hab voorziet hierin.
+  - De was-wordt-lijst die gebruikt wordt in veg2hab is direct bruikbaar voor veg2leef, omdat deze uitsluitend wordt gebruikt voor het vertalen tussen landelijke typologieën.
+- `stap 2` uit veg2hab kan zonder aanpassingen gebruikt worden om vegetatiekarteringen samen te voegen voordat de LG-typen bepaald worden.
+- Voor het omzetten van vegetatiekaarten naar LG-typenkaarten, kan met een klein aanpassing `stap 3` gebruikt worden:
+  - Er moet een **definitietabel LG-typen** gemaakt worden, in dezelfde vorm als de definitietabel habitattypen: regels die omschrijven welke vegetatietypen leiden tot welk LG-type, aangevuld met criteria met betrekking tot Fysisch Geografische Regio's. Al deze informatie is beschikbaar in het Methodendocument;
```

Comment:
Misschien kun je hier of hierboven op regel 25 nog even expliciet benoemen dat alle code om FGR criteria te checken al aanwezig is, en dat daar niks meer aan hoeft te gebeuren.